Jenkinsfile

```diff
@@ -11,7 +11,8 @@ def wrapStep(String stepName, Closure step) {
 node('insights-frontend-slave') {
   if ('master' == env.BRANCH_NAME) {
     wrapStep('clone', { name -> stage(name) { checkout scm } })
-    wrapStep('deploy_dashboard', { name -> stage(name) { sh 'rsync -arv -e "ssh -2" * sshacs@unprotected.upload.akamai.com:/114034/insights/platform/dashboard/' } })
+    wrapStep('deploy_dashboard', { name -> stage(name) { sh 'rsync -arv -e "ssh -2" * sshacs@unprotected.upload.akamai.com:/114034/insights/static/chrome/' } })
+    wrapStep('deploy_dashboard', { name -> stage(name) { sh 'rsync -arv -e "ssh -2" * sshacs@unprotected.upload.akamai.com:/114034/insightsbeta/static/chrome/' } })
```

Comment:
Do we want to change `deploy_dashboard` to something like `deploy_chrome`/`deploy_chrome_beta`?